render/wgpu/src/lib.rs

```diff
@@ -186,33 +185,27 @@ pub enum QueueSyncHandle {
 impl SyncHandle for QueueSyncHandle {
     fn retrieve_offscreen_texture(
         self: Box<Self>,
-    ) -> Result<ruffle_render::bitmap::Bitmap, ruffle_render::error::Error> {
-        let image = self.capture();
-        Ok(Bitmap::new(
-            image.dimensions().0,
-            image.dimensions().1,
-            ruffle_render::bitmap::BitmapFormat::Rgba,
-            image.into_raw(),
-        ))
+        with_rgba: RgbaBufRead,
+    ) -> Result<(), ruffle_render::error::Error> {
+        self.capture(|buf, buf_width| with_rgba(buf, buf_width));
```

Comment:
Can this be just `self.capture(with_rgba`)` ?

Reply:
Fixed, good catch!